server/src/main/models/index.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs: any        = require('fs');
+var path: any      = require('path');
+var Sequelize: any = require('sequelize');
+var basename: string  = path.basename(__filename);
+var env: string       = process.env.NODE_ENV || 'development';
+var config: any    = require(__dirname + '/..\..\..\src\main\config\config.js')[env];
+var db: any        = {};
+
+if (config.use_env_variable) {
+  var sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  var sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
+  .filter((file: string): boolean => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
```

Comment:
Double quotes. Actually, this applies to the whole file. Don't we have TSLint set up to detect this?